extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/binder/mpmetrics/AnnotationHandler.java

```diff
@@ -81,7 +81,7 @@ static boolean removeCountedWhenTimed(DotName sourceAnnotation, AnnotationTarget
                         findAnnotation(classInfo.classAnnotations(), MetricDotNames.SIMPLY_TIMED_ANNOTATION) == null) {
                     return false;
                 }
-                log.warnf("Bean %s is both counted and timed. The @Counted annotation " +
+                log.infof("Bean %s is both counted and timed. The @Counted annotation " +
```

Comment:
Why?

Reply:
So to clarify, I think we should keep them `warn`. If you don't want these log messages every time (bad UX), you need to do something about it thus the warn.

Reply:
@loicmathieu suggested the warning was too strong (thinking something was wrong). As you and I discussed, I made it a warning to begin with because the behavior can be unexpected (in terms of metrics being emitted).

I'm reconsidering. ;) I'll drop this commit, I think.

Reply:
Warning means something went wrong but we recover from this. 

If I use open metrics, as per the spec I must add the two annotations to have the two metrics. 
For interoperability concerns keeping the two annotations is a valid choice.

So this message is an information that micrometer is compatible with open metrics but didn't follow the spec.

I would change it to debug level after a second thinking. 

But I'm ok to keep warning it you didn't agree with the way I see this :)